src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction.',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {},
```

Comment:
Hi @1GORDON. Should I do it just with `return state.status`? Because as I understand if I do that return the state will update and replace the inicial state.
Thanks for your time by the way!